apps/console/src/features/saml2-configuration/pages/saml2-configuration.tsx

```diff
@@ -0,0 +1,392 @@
+                                    </Placeholder.Header>
+                                </Placeholder>
+                                <Divider hidden />
+                            </div>
+                        </div>
+                        <div className="content extra extra-content">
+                            <div className="action-button">
+                                <Placeholder>
+                                    <Placeholder.Line length="very short" />
+                                </Placeholder>
+                            </div>
+                        </div>
+                    </div>
+                    <Divider hidden/>
+                </div>
+            </Grid.Row>
+        );
+    };
+
+    return (
+        <PageLayout
+            title={ t("console:saml2Config.title") }
+            pageTitle={ t("console:saml2Config.title") }
+            description={ t("console:saml2Config.description") }
+            backButton={ {
+                onClick: () => onBackButtonClick(),
+                text: t("console:manage.features.governanceConnectors.goBackLoginAndRegistration")
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ false }
+            pageHeaderMaxWidth={ true }
+            data-componentid={ `${ componentId }-form-layout` }
+        >
+            <Ref innerRef={ pageContextRef }>
+                <Grid className={ "mt-2" } >
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 16 }>
+                            <EmphasizedSegment className="form-wrapper" padded={ "very" }>
+                                { isSaml2FetchRequestLoading
+                                    ? renderLoadingPlaceholder() 
+                                    : (
+                                        <>
+                                            <Form
+                                                id={ FORM_ID }
+                                                uncontrolledForm={ true }
+                                                onSubmit={ handleSubmit }
+                                                initialValues={ saml2Config }
+                                                enableReinitialize={ true }
+                                                ref={ formRef }
+                                                validate={ validateForm }
+                                                autoComplete="new-password"
+                                            >
+                                                <Grid>
+                                                    <Grid.Row columns={ 1 } key={ 3 }>
+                                                        <Grid.Column width={ 10 }>
+                                                            <Field.Checkbox
+                                                                ariaLabel="Enable metadata signing checkbox"
```

Comment:
Shall we use the i18n for aria labels as well?

Reply:
Fixed with 1d3ef8fd4f615de27060c959ea91a56ca1f3158f